generators/app/index.js

```diff
@@ -84,6 +84,22 @@ module.exports = yeoman.Base.extend({
           }
         ]
       },
+      // jade
+      {
+        type: 'list',
+        name: 'jade',
+        message: 'Choose HTML or JADE',
+        choices: [
+          {
+            name: 'classical HTML',
```

Comment:
I'd prefer the phrases 'HTML Templating (recommended)' and 'Jade Templating', as it's easier for beginners and many developers prefer HTML over Jade. And for now it's just a smaller technology stack, which makes it easier to begin with.